### PR TITLE
feat(github-bot): refactor comment + add force skip

### PR DESCRIPTION
**Pull request by aeddi**
This PR significantly modifies the github-bot's comment and adds a button to force the success of its CI check, even it the requirements provided in the config are not met.

Related to https://github.com/gnolang/gno/issues/3238#issuecomment-2526174402

**Edit**: I updated [the comment below](https://github.com/gnolang/gno/pull/3311#issuecomment-2528477336) by running the bot on my laptop if you want to see the result (so the skip button is not working yet).

### CODE REVIEW

**Comment by Gno2D2** [comment]
I'm a bot that assists the Gno Core team in maintaining this repository. My role is to ensure that contributors understand and follow our guidelines, helping to streamline the development process.

The following requirements must be fulfilled before a pull request can be merged.
Some requirement checks are automated and can be verified by the CI, while others need manual verification by a staff member.

These requirements are defined in this [configuration file](https://github.com/gnolang/gno/tree/master/contribs/github-bot/internal/config/config.go).

## Automated Checks

 🟢 Maintainers must be able to edit this pull request ([more info](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork))
 🔴 The pull request head branch must be up-to-date with its base ([more info](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch))


## Manual Checks

*No manual checks match this pull request.*

<details><summary><b>Debug</b></summary><blockquote>
<details><summary><b>Automated Checks</b></summary><blockquote>

<details><summary>Maintainers must be able to edit this pull request (more info)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 The pull request was created from a fork (head branch repo: aeddi/gno)

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Maintainer can modify this pull request

```
</blockquote></details>

<details><summary>The pull request head branch must be up-to-date with its base (more info)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🔴 Requirement not satisfied
└── 🔴 Head branch (aeddi:dev/aeddi/github-bot-refactor) is up to date with base (master): behind by 4 / ahead by 1

```
</blockquote></details>

</blockquote></details>



</blockquote></details>

**Review by thehowl — APPROVED** [review]

**Pull request merged**
